tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -508,7 +508,7 @@ Status StorageManager::array_consolidate(
     uint32_t key_length,
     const Config* config) {
   // Check array URI
-  URI array_uri(array_name);
+  URI array_uri = make_uri(array_name, config);
```

Comment:
Any particular reason an assignment is used rather than just constructing with make_uri() as constructor argument as was done previously with array_name? (I didn't look, maybe there's not a constructor taking string, but then could use .c_str())

Reply:
removed make_uri, since we don't know the requirement of the customer. at the mean time, it might be better to minimize the changes. thanks for the comment